pkg/sshd/server_test.go

```diff
@@ -98,20 +110,20 @@ func TestPushInvalidArgsLength(t *testing.T) {
 	key, err := sshTestingHostKey()
 	assert.NoErr(t, err)
 
-	cfg := ssh.ServerConfig{NoClientAuth: true}
+	cfg, _ := serverconfigure()
```

Comment:
make sure to check the returned `error` here (I know it's not likely to ever be non-nil, but might change in the future)